lms/views/admin/organization.py

```diff
@@ -80,6 +80,12 @@ def new_organization_callback(self):
         renderer="lms:templates/admin/organization/show.html.jinja2",
         permission=Permissions.STAFF,
     )
+    @view_config(
+        route_name="admin.organization.section",
+        request_method="GET",
+        renderer="lms:templates/admin/organization/show.html.jinja2",
+        permission=Permissions.STAFF,
+    )
```

Comment:
I'm not sure what's the best way to avoid duplicating all params which are the same on this `view_config` and the one above.

On previous PR I defined a `view_defaults` at class level, but this class has more views.

Reply:
I don't think it's that important, the duplication helps understand what's going on here better anyway